utils/debugger/src/check-order.js

```diff
@@ -20,8 +20,7 @@ const checkOrder = async (order, network) => {
 
   // Check the order has all necessary fields
   if (!orders.isValidOrder(order)) {
-    errors.push('Order structured incorrectly')
-    return errors
+    errors.push('Order structured incorrectly or signature invalid')
```

Comment:
add push errors to line 27 as well for consistency.

Reply:
Do you mean to remove the return?

Reply:
yes if the intended behavior is to proceed to the end and return all the errors.